src/views/FAQ/FAQ.tsx

```diff
@@ -267,10 +269,11 @@ export const FAQPage: FunctionComponent = () => (
           <FAQItem question="Why isn’t my package displayed on Construct Hub?">
             <Text>
               Construct Hub includes publicly-available constructs that are
```

Comment:
"How can I report a package?" Should be replaced with "How can I report abuse?" - and the answer should also be updated with click "Report abuse" instead of "Report package"